cfme/tests/configure/test_config_manual.py

```diff
@@ -32,3 +32,39 @@ def test_validate_landing_pages_for_rbac():
             1. Landing pages which user has access to must be present in the dropdown list.
     """
     pass
+
+
+@pytest.mark.manual
+@test_requirements.settings
+@pytest.mark.tier(1)
+def test_my_settings_default_views_alignment():
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/20h
+        testSteps:
+            1. Go to My Settings -> Default Views
+            2. See that all icons are aligned correctly
```

Comment:
@valaparthvi  do you plan to add some `expectResults` here after this is merged?

Reply:
@john-dupuy let me handle it :)